charts/piraeus/README.md

```diff
@@ -3,33 +3,24 @@
 Deploys the [Piraeus Operator](https://github.com/piraeusdatastore/piraeus-operator) which deploys and manages a simple
 and resilient storage solution for Kubernetes.
 
-The main deployment method for Piraeus Operator switched to [`kustomize`](../../docs/tutorial)
+The main deployment method for Piraeus Operator switched to [`kustomize`](https://piraeus.io/docs/stable/tutorial/get-started/)
 in release `v2.0.0`. This chart is intended for users who want to continue using Helm.
 
 This chart **only** configures the Operator, but does not create the `LinstorCluster` resource creating the actual
-storage system. Refer to the existing [tutorials](../../docs/tutorial)
-and [how-to guides](../../docs/how-to).
+storage system. Refer to the [how-to guide](https://piraeus.io/docs/stable/how-to/helm/).
 
 ## Deploying Piraeus Operator
 
 To deploy Piraeus Operator with Helm, clone this repository and deploy the chart:
 
 ```
-$ git clone --branch v2 https://github.com/piraeusdatastore/piraeus-operator
-$ cd piraeus-operator
-$ helm install piraeus-operator charts/piraeus-operator --create-namespace -n piraeus-datastore
+$ helm upgrade --install --create-namespace --namespace piraeus-datastore piraeus-operator oci://ghcr.io/piraeusdatastore/piraeus-operator/piraeus --set installCRDs=true --wait
```

Comment:
How about removing the "Deploying Piraeus Operator" section entirely to reduce duplication? We already refer to the how-to guide which contains all the necessary information.